library/HelloSign/EmbeddedSignatureRequest.php

```diff
@@ -52,11 +52,10 @@ public function toParams()
          * our defaults.
          */
         return $this->request->toParams(array(
-            'except' => array('allow_decline')
+            
```

Comment:
might consider keeping the except key and passing an empty array to that for future exceptions?